### PR TITLE
Ab1797 cypress tests

### PR DESCRIPTION
**Pull request by matt-paul**
[Ticket]()

[Zeplin]()

## Pre-review checklist:

- [x] Tests

## Test:

- [x] Tested locally

- Platforms/Browsers:

  - [x] Google Chrome (latest version)

The introduction of data-cy is a Cypress Best Practise:
https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements

### CODE REVIEW

**Comment by declanslevin** [comment]
The :focus/:active states for the "Delete file" menu item are missing, at the moment it just shows the browser default behaviour which is a blue outline. It should have the yellow background and black border-bottom.

File: frontend/cypress/integration/frontend/workspaces/folders/files/delete-file.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    cy.visit(start);
+    cy.contains("h1", "London Region NHS England Safeguarding Annual Review");
+
+    cy.contains("Delete file").should("not.exist");
+    cy.get(`[data-cy="file-options"]`).click();
+    cy.contains("Delete file").should("exist");
+
+    cy.contains("Files").should("not.exist");
+    cy.get(`[data-cy="delete-file"]`).click();
+    cy.contains("Files").should("exist");
```
Comment (frigus02) [inline frontend/cypress/integration/frontend/workspaces/folders/files/delete-file.js:13]: It's not super obvious to me what this check for. Does this check that we are redirected to the folder page after deleting a file? In this case, maybe checking for the URL would be more obvious. E.g.:

```
cy.location("pathname").should("eq", "/workspaces/750a5fb7-47da-4a8c-8d2e-399ae8d24cde/folders/63c7f15a-7077-4b46-8808-99227ad06c8f");
```

File: frontend/components/Menu/MenuListItem.tsx [hunk 2]
```diff
@@ -55,6 +58,26 @@ const StyledListItem = styled(ListItem)`
   }
 
   ${({ theme }) => `
+   button {
+     &:hover {
+        opacity: 1;
```
Comment (frigus02) [inline frontend/components/Menu/MenuListItem.tsx:63]: I think opacity might not be needed. It doesn't seem to make any difference if I remove it.

**Review by frigus02 — APPROVED** [review]

File: frontend/components/Menu/MenuListItem.tsx [hunk 3]
```diff
@@ -55,6 +58,26 @@ const StyledListItem = styled(ListItem)`
...
+        background-color: ${theme.colorShadeNhsukBlue35};
+      }
+      &:active, :focus {
+        color: ${theme.colorNhsukBlue};
+        &.hidden {
+          color: ${theme.colorNhsukBlack};
+        }
+        border:none;
+        background-color: ${theme.colorNhsukYellow};
+        box-shadow: 0 -2px #ffeb3b, 0 4px #212b32;
```
[the remark this answers is not in the document]
Comment (declanslevin) [reply frontend/components/Menu/MenuListItem.tsx:77]: Screenshot 2020-10-20 at 14 18 55

[the remark this answers is not in the document]
Comment (declanslevin) [reply frontend/components/Menu/MenuListItem.tsx:77]: Although this also happens on other menu items elsewhere, so might be something I can just log as a bug in general

**Review by declanslevin — APPROVED** [review]

**Pull request merged**